src/renderer/windows/settings/Settings.vue

```diff
@@ -313,6 +326,14 @@ window.ytmd.handleUpdateDownloaded(() => {
             @clear="removeCustomCSSPath"
           />
           <YTMDSetting v-model="zoom" type="range" max="300" min="30" step="10" name="Zoom" @change="settingsChanged" />
+          <YTMDSetting
+            v-if="isLinux"
+            v-model="trayIconStyle"
+            :options-map="toOptionsMap(trayIconStyles)"
```

Comment:
Nit: This should capitalize the options for the UI. The actual config doesn't need it.